app/models/school.rb

```diff
@@ -32,4 +32,23 @@ def sierra_code
 
     return match.sierra_code
   end
+
+  
+  # Get all active schools
+  def self.active_schools
+    active.map do |school|
+      [school.name + school_code(school), school.id] 
```

Comment:
We are repeating that #{school.name + school_code(school)} construction, maybe you could make a new instance method called, say, `label`, which would combine name and code, and could also add the [INACTIVE] label if the school isn't active. What do you think?

Reply:
After that, we are also using the `[school.label, school.id]` construction a lot. So maybe that could go into an instance method called, say, `label_and_id`. That would be used here and in the settings controller. Please let me know what you think — just suggestions!